staking/programs/integrity-pool/Cargo.toml

```diff
@@ -25,3 +25,8 @@ litesvm = "0.1.0"
 solana-program = { workspace = true }
 solana-sdk = { workspace = true }
 spl-token = "4.0.1"
+wormhole-vaas-serde =  "0.1.0"
+wormhole-solana-vaas = {version="0.3.0-alpha.1", features = ["anchor", "encoded-vaa", "mainnet"]}
+serde_wormhole = "0.1.0"
+pythnet-sdk = {git= "https://github.com/pyth-network/pyth-crosschain.git", branch = "feat/publisher-caps-program" , features = ["test-utils"]}
```

Comment:
do we need to change this after merging the other PR? Since the branch will be deleted.

Reply:
done